pyanaconda/modules/payloads/payload/payload_base_interface.py

```diff
@@ -78,6 +78,9 @@ def DefaultSourceType(self) -> Str:
 
         :return: a string representation of a source type
         """
+        if self.implementation.default_source_type is None:
```

Comment:
If this is a special case for now, maybe we should log a warning ? Also maybe a comment on what this means, as even with the commit message it is not really apparently why returning "" is fine here. :)